packages/replay-next/pages/variables.css

```diff
@@ -2,6 +2,108 @@
   --font-family-default: Inter, sans-serif;
   --font-family-monospace: Menlo, ui-monospace, SFMono-Regular, Monaco, Consolas, Liberation Mono,
     Courier New, monospace;
+
+  /* Migration note: everyone below this was removed from src/devtools/client/themes/variables.css */
```

Comment:
These variables look like they're in the wrong place. They're mostly _colors_ but they're outside of a light/dark theme block.

Colors should all be within either the `:root.theme-dark` or `:root.theme-light` block.